src/Internal/DependencyManagement/ThirdPartyServiceProvider.php

```diff
@@ -73,5 +73,17 @@ function () {
 					->getValidator();
 			}
 		);
+
+		// Update Jetpack connection with a translatable name, after init is called.
+		add_action(
+			'init',
+			function () {
+				$manager = $this->getLeagueContainer()->get( Manager::class );
+				$manager->get_plugin()->add(
+					__( 'Google for WooCommerce', 'google-listings-and-ads' ),
+					[]
```

Comment:
I think we can skip the last argument completely
```suggestion
					__( 'Google for WooCommerce', 'google-listings-and-ads' )
```